setup.py

```diff
@@ -19,6 +19,7 @@
     "PGPy",
     "python-jose<=0.5.6",
     "redo",
+    "toposort==1.0",
```

Comment:
Any reason we're not using the 1.5 latest version https://pypi.python.org/pypi/toposort ?

Reply:
Good point. Probably it was 1.0 when I started. :)